python/tests/graph/test_utils.py

```diff
@@ -27,66 +27,135 @@
 TestCase = namedtuple('TestCase', ['data', 'description'])
 
 
-def _gen_graph_elems_names():
+def _gen_tensor_op_string_input_tests():
     op_name = 'someOp'
-    for tnsr_idx in range(17):
+    for tnsr_idx in [0, 1, 2, 3, 5, 8, 15, 17]:
         tnsr_name = '{}:{}'.format(op_name, tnsr_idx)
         yield TestCase(data=(op_name, tfx.op_name(tnsr_name)),
-                       description='must get the same op name from its tensor')
+                       description='test tensor name to op name')
         yield TestCase(data=(tnsr_name, tfx.tensor_name(tnsr_name)),
-                       description='must get the tensor name from its operation')
+                       description='test tensor name to tensor name')
 
 
-def _gen_wrong_graph_elems_types():
+def _gen_invalid_tensor_or_op_input_with_wrong_types():
     for wrong_val in [7, 1.2, tf.Graph()]:
         yield TestCase(data=wrong_val, description='wrong type {}'.format(type(wrong_val)))
 
 
-def _gen_graph_elems():
+def _gen_valid_tensor_op_objects():
```

Comment:
now it seems to be testing all input combos. if so, rename to `_gen_valid_tensor_op_input_combos`